config/routes.rb

```diff
@@ -29,6 +29,14 @@
     end
   end
 
+  resources :bookings, only: [:index, :create, :destroy] do 
+    post :accept
+  end
+
+  resources :invites, only: [:index, :create, :destroy] do 
+    post :accept
+  end
```

Comment:
For these additional `accept` routes (lines 33 and 37), consider making use of the `on: :member` option: http://guides.rubyonrails.org/routing.html#adding-member-routes

Its only a minor thing, but it'll mean you can use `params[:id]` consistently throughout the resource controller actions, instead of having to use `params[:invite_id]` only in the `accept` action.